contracts/examples/dao/DAO.sol

```diff
@@ -54,28 +56,20 @@ contract DAO is VentureEth {
         revert("Withdraw is disabled.");
     }
 
-    /** Venture investment */
-
     /**
-     * @notice Propose a venture. Must have approved the DAO to spend gage of your DAO tokens. venture must inherit from VentureEth.
-     * @param venture The address of the VentureEth contract
-     * @param investment The ether to invest in the venture.
+     * @notice To be called during the first investment round.
      */
-    function proposeVenture(
-        address venture,
-        uint256 investment
-    ) public {
-        // Maybe use ERC165 to make sure venture it's a VentureEth
-        require(currentState == "LIVE", "DAO needs to be LIVE");
-        Voting voting = new Voting(
-            address(this),
-            address(this),
-            abi.encodeWithSignature("investVenture(address,uint256)", venture, investment),
-            threshold);
-        proposals.add(address(voting));
-        emit VentureProposed(address(voting));
+    function startDistribution() public onlyOwner {
+        // solium-disable-next-line security/no-low-level-calls
+        (bool success, ) = address(this).delegatecall(
```

Comment:
Please remind me again why can't we just call `transferOwnership(address(this))`

Reply:
Here's why: when you call `transferOwnership(address(this))`, the `msg.sender` in the transfer call becomes `address(this)`, not `owner` (which was the one who initiated the call to `startDistribution`). Therefore, you have to delegate the call, that is, call with `msg.sender = tx.origin = owner`.

Reply:
I get it now, thanks!